apps/calendar/locales/calendar.en-US.properties

```diff
@@ -42,7 +42,7 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+modify-account-setting-up=Setting up account
```

Comment:
I think we need to change this string before landing. When changing the value of strings, we also need to update the keys to inform our localizers that something has changed.